render/interfaceFeatures.go

```diff
@@ -0,0 +1,18 @@
+package render
```

Comment:
Maybe eventually, but it'd probably be a subpackage of render? If it's interfaces for behaviors that some renderables support and some don't